dev.py

```diff
@@ -52,8 +52,11 @@ def install_pip():
     tmp = tempfile.mkdtemp(prefix='buildout-dev-')
     try:
         get_pip = os.path.join(tmp, 'get-pip.py')
+        GET_PIP_URL = 'https://bootstrap.pypa.io/get-pip.py'
+        if sys.version_info < (3, ):
+            GET_PIP_URL = 'https://bootstrap.pypa.io/2.7/get-pip.py'
         with open(get_pip, 'wb') as f:
-           f.write(urlopen('https://bootstrap.pypa.io/get-pip.py').read())
+            f.write(urlopen(GET_PIP_URL).read())
```

Comment:
Minor:

```suggestion
            with urlopen(GET_PIP_URL) as r:
                f.write(r.read())
```

Reply:
Thanks! I learned something new.

Reply:
Uhm, this made boom on Python 2.7 - I just pushed the old version.

Reply:
Ahh oops.